refl1d/probe.py

```diff
@@ -1041,7 +1041,7 @@ def load4(filename, keysep=":", sep=None, comment="#", name=None,
           theta_offset=0, sample_broadening=0,
           L=None, dL=None, T=None, dT=None,
           FWHM=False, radiation=None,
-          columns=None,
+          columns=None,data_range=[None,None],
```

Comment:
Lint will complain about spacing and about mutable default arguments.  Use the following instead:
```
columns=None, data_range=(None, None),
```